### DISCUSSION TITLE
4.28.2的病毒误杀

### DISCUSSION THREAD

**Discussion by 782727697** [MaaAssistantArknights/MaaAssistantArknights#7619] [Q&A] [answered]
4.28.2的windows release包自动更新后MAA.exe被windows当成病毒杀掉了，这是正常情况吗？
![图片](https://github.com/MaaAssistantArknights/MaaAssistantArknights/assets/10194529/320047ad-7e63-4839-a299-540c84a5a272)

**Comment by ABA2396** [comment, marked as answer]
#6662
